### DISCUSSION TITLE
Persist context?

### DISCUSSION THREAD

**Discussion by aantix** [rubycdp/ferrum#205] [Q&A]
Is there any way to save a context? 

So that I can programmatically login, and then the next time I open the browser, the user should still be logged in?

**Comment by route** [comment]
I think you can try to pass `user-data-dir` flag to `browser_options`

**Reply by Nakilon** [reply]
I ran it with browser_options: `{"user-data-dir" => "profile"}`, authorized in discord, quit, then after I ran the script again it asked the login/password like it the cookie or localstorage  is somehow not working.

**Reply by elthariel** [reply]
Not really helpful, but for the sake of cross-reference: #47
